src/App.js

```diff
@@ -96,6 +96,7 @@ function App(props) {
 
   return (
     <div className="todoapp stack-large">
+      <h1>Todo</h1>
```

Comment:
I found that in the content repo, we use `TodoMatic` 😄 so let's use that instead:

```suggestion
      <h1>TodoMatic</h1>
```

https://github.com/mdn/content/blob/eb384d1cb58a240bc0d46545fffae7e1218953a6/files/en-us/learn/tools_and_testing/client-side_javascript_frameworks/react_todo_list_beginning/index.md?plain=1#L90